### ISSUE TITLE
Replace:  newline character \n, possibley other hidden caracters are ignored

### ISSUE THREAD

**Issue by revast** [notepadqq/notepadqq#457]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
I want to delete new lines in a text file.  Selecting a new line does not show up the hidden character  \n, hence I cannot replace it in the text file.

**Expected behavior**
\n character shows up, and can be replaced like any other character

**Minimal reproduction of the problem with instructions**
In gedit, character \n appears in the Replace dialog when selecting text over a line.
In npqq, the newline characrer gets ignored, so for instance it is not possible to delete all new lines in a file, to have just one line

**Please tell us about your environment:**
Ubuntu 16.04, Qt 5.**6**, npqq from PPA latest.

**Comment by JuBan1** [comment]
Hi, thanks for the report!

The editor implementation doesn't support matching across lines which includes newline characters. The new search panel (#446) can do that however. Hopefully we'll get it in soon.

(See also #60.)

**Comment by JuBan1** [comment]
Fixed in de15c09 using the new search and replace tool.
